src/test/kotlin/sirius/kernel/async/BackgroundLoopTest.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.async
+
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
+import sirius.kernel.commons.Wait
+import kotlin.test.assertTrue
+
+@Tag("nightly")
```

Comment:
**Question:** Shouldn't this be replaced by `@NightlyTest`?

Reply:
yes, thanks, just updated that!